### ISSUE TITLE
PoeHud is not working

### ISSUE THREAD

**Issue by omegasrevenge** [TehCheat/PoEHUD#99]
When downloading the new version and opening the executable I get a popup "Game is not running", even though it obviously is, right behind the popup itself. When trying to use an older version of the app it just spits out a very long error code in another popup. 
What information do you need from me to fix it, or is a fix already underway?

**Comment by TehCheat** [comment]
Use the right version of HUD for the version of PoE you're running.  The master branch is for x86/DirectX 9, the x64 branch is for x64/DirectX 11.

**Comment by omegasrevenge** [comment]
Edit#2: I think I've found the problem. I am downloading the latest files from the release list, but those do not actually contain the latest version of the app. I think. Could the release please get updated if this is the issue?

**Comment by TehCheat** [comment]
Yeah, the release version is for 2.5.2b, so no surprise it's not working with 2.6.0.  I'll try and get that updated.

**Comment by omegasrevenge** [comment]
Thank you for your quick response! This answer everything and I just got it to work.
